site/content/docs/contribute/roles-and-governance.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Roles and Governance"
+type: page
+cascade:
+  EditableContent: true
+  addBreadcrumbs: true
+---
+
+ZAP's governance model is simple.
+
+There are two roles: general contributors, and the [Core Team](/docs/desktop/credits/).
+
+Decisions within the project are made based on majority rules. That is to say, if someone puts forth a new idea or decision to be made, it is made based on the majority of the team agreeing on the direction/outcome.
```

Comment:
Maybe tweak to say that anyone can work on anything, but it a majority Core Team decision as to whether significant new functionality is adopted (and therefore supported) by the Core Team. All PRs must be approved by 2 members of the Core Team, not including the submitter?

Reply:
Sure